test/Libraries/RevitIntegrationTests/ElementTests.cs

```diff
@@ -58,6 +58,26 @@ public void CanGetElementSubComponents()
             Assert.IsNull(resultWallSubElement);
         }
 
+        [Test]
+        [TestModel(@".\Element\elementComponents.rvt")]
+        public void CanGetElementSuperComponent()
+        {
+            // Arrange
+            string samplePath = Path.Combine(workingDirectory, @".\Element\canGetElementSuperComponent.dyn");
+            string testPath = Path.GetFullPath(samplePath);
+            int expectedWindowSuperComponentId = 319481;
```

Comment:
use `parent` instead of `super` now, would make it easier to understand code when reading